### PR TITLE
style(replays): adjusted display of tooltip content so it wraps better

### LINKED ISSUE #35680: Timeline tooltips overflow when urls/content are long
Example url: https://dev.getsentry.net:7999/organizations/sentry-emerging-tech/replays/sourcemapsio-replays:75abb7e4486145c89ed2ba1cfb09ca75/

1. Broken tooltip, content overflows the tooltip container and is hard to read:

Screen Shot 2022-06-16 at 4 02 17 PM
(note: screenshot updated after https://github.com/getsentry/sentry/pull/35713 was merged)

2. In the sidebar the same content is wrapping and is readable.

Image

### PR DESCRIPTION
**Pull request by Danondso**
Closes #35680 
Set a max width on crumbdetails so it doesn't bleed past the tooltip.
## Screenshots
Here's a smattering of varying messages with the fix in place.
image
image
image
image

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx [hunk 1]
```diff
@@ -112,8 +112,10 @@ const HoverList = styled('ul')`
   margin: 0;
   padding: 0;
 `;
-const HoverListItem = styled('li')`
-  display: flex;
+
+const HoverListItem = styled('ul')`
+  display: grid;
+  grid-template-columns: max-content 150px;
```
Comment (billyvg) [inline static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx:118]: Could we try 250px?

Comment (Danondso) [reply static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx:118]: It bleeds over 
image

Here it is at 175
image

Comment (evanpurkhiser) [reply static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx:118]: That seems like it could bleed over either way depending on the content.

Comment (ryan953) [reply static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx:118]: @Danondso can you try `225px`? (per [tooltip.tsx](https://github.com/getsentry/sentry/blob/2beeba84e9e9c8e8d4bb0b397ea5356d32868d25/static/app/components/tooltip.tsx#L392))

That 225px is less than the hovercard max of [295px](https://github.com/getsentry/sentry/blob/2beeba84e9e9c8e8d4bb0b397ea5356d32868d25/static/app/components/hovercard.tsx#L316), but the linebreaks are what we really need here.


@evanpurkhiser The change we need to make is to force linebreaks in the content, and have the popup be as wide as we can get. Is there another component combo that might this work?

**Comment by ryan953** [comment]
We should make sure to rebase this ontop of https://github.com/getsentry/sentry/pull/35713 and re-test it. 

It looks like the message format inside `replayTimelineEvents.tsx` got updated already

File: static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx (hunk 1 above)
Comment (Danondso) [reply static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx:118]: @ryan953 225px lookin' sad 
image

File: static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx [hunk 2]
```diff
@@ -109,8 +109,10 @@ const HoverList = styled('ul')`
   margin: 0;
   padding: 0;
 `;
-const HoverListItem = styled('li')`
-  display: flex;
+
+const HoverListItem = styled('ul')`
```
Comment (ryan953) [inline static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx:113]:
```suggestion
const HoverListItem = styled('li')`
```

**Comment by ryan953** [comment]
well we got the wrapping, so it's strictly better than before.

**Review by ryan953 — APPROVED** [review]

**Pull request merged**